Tests/XCUITests/PhotonActionSheetTests.swift

```diff
@@ -75,7 +75,7 @@ class PhotonActionSheetTests: BaseTestCase {
         }
         mozWaitForElementToExist(fennecElement, timeout: 5)
         fennecElement.tap()
-        mozWaitForElementToExist(app.navigationBars["ShareTo.ShareView"], timeout: TIMEOUT)
+        mozWaitForElementToExist(app.navigationBars["ShareTo.ShareView"])
```

Comment:
Not sure why removing the timeout will work better... usually we need the timeout

Reply:
There is a default timeout on the definition of the mozWaitForElementToExist function.